### PR TITLE
Added Onfido iOS SDK with Swift

### PR DESCRIPTION
**Pull request by ngo275**
## Overview

I added iOS SDK to RN app with Swift not Obj-C.

### CODE REVIEW

File: ios/ReactNativeSampleApp/OnfidoSDK.swift [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+                      rejecter reject: @escaping RCTResponseSenderBlock) -> Void {
+    DispatchQueue.main.async {
+      self.run(withApplicationID: applicantID, resolver: resolve, rejecter: reject)
+    }
+  }
+  
+  private func run(withApplicationID id: String,
+                   resolver resolve: @escaping RCTResponseSenderBlock,
+                   rejecter reject: @escaping RCTResponseSenderBlock) {
+    let token = "ONFIDO_TOKEN"
```
Comment (ngo275) [inline ios/ReactNativeSampleApp/OnfidoSDK.swift:25]: To test,  this should be replaced to real one.

Comment (huangmr) [reply ios/ReactNativeSampleApp/OnfidoSDK.swift:25]: super, when will this be merged please?

**Comment by rfreitas** [comment]
First skim, it looks great. I will add a task for one of our developers review it and have a tester accept it. The PR coincided with our beginning of the sprint, so it will likely get merged beginning of next month.

**Review by rfreitas — APPROVED** [review]

**Comment by rfreitas** [comment]
Sorry for the massive delay.
I tested on xcode 9.4 and iphone 6. Worked!
I also tested on an Android device and it still worked.

**Comment by rfreitas** [comment]
Thanks for the contribution! Very much appreciated :)

**Pull request merged**